BootloaderCommonPkg/Library/PagingLib/Ia32/Paging32.nasm

```diff
@@ -77,18 +77,18 @@ ASM_PFX(DisablePaging32):
 global ASM_PFX(FlushCacheLine)

```

Comment:
This FlushCacheLine can be removed now since we have AsmFlushCacheRange in ExtraBaseLib.  That should be used instead.

Reply:
Okay. Let me remove it.